no_cluster.robot

```diff
@@ -0,0 +1,79 @@
+*** Settings ***
```

Comment:
Can we call this file just repos.robot? There might be more repo stuff to test, and also alot more non-cluster stuff that can pushed into other files

Reply:
As you deduced, I used that name in preparation to add tests for other non-cluster commands. But I like splitting those into separate files instead. I will rename.

Thanks for all the reviews!